src/Ontology/Oep4.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Oep4.h"
+#include <HexCoding.h>
+#include <cstdio>
+
+using namespace TW;
+
+namespace TW::Ontology {
+Oep4::Oep4(const Address addr) noexcept
+    : oep4Contract(addr._data.begin(), addr._data.end()) {
+}
+
+Transaction Oep4::romethod0(std::string method_name, uint32_t nonce) {
```

Comment:
The `romethod0` looks typo to me, more readable name?

Reply:
It means (oep4) read only method with 0 argument. Suggest a better name?

Reply:
either `readonlyMethod` or add a comment, hard to guess what it is

Reply:
OK, helper method name changed